packages/farrow-schema/src/helper.ts

```diff
@@ -243,3 +273,78 @@ export const partial: typeof partialStruct & typeof partialObject = (
 
   throw new Error(`Unknown Schema Constructor: ${Ctor}`)
 }
+
+const getRequiredFields = (fields: FieldDescriptors): FieldDescriptors => {
+  const descriptors = {} as FieldDescriptors
+
+  for (const [key, value] of Object.entries(fields)) {
+    if (isFieldDescriptor(value)) {
+      descriptors[key] = isNullableType(value) ? (value as any).Item : value
```

Comment:
`value` is the constructor of Nullable subclass, not an instance of it, so there is no `Item` field in it.

try `getInstance(value).Item` to see the type-check result.